src/main/java/gregtech/api/util/GTUtility.java

```diff
@@ -1113,4 +1140,51 @@ public static boolean tryBreakSnowLayer(World world, BlockPos pos, @Nonnull IBlo
         }
         return false;
     }
+
+    /**
+     * Creates a list of biomes whose weight, as described by the passed Function, differs from the passed original weight
+     * For use in the JEI Ore Spawn Page and JEI Fluid Spawn Page
+     *
+     * @param biomeFunction A Function describing the modified weights of biomes
+     * @param originalWeight The original weight of the biome
+     *
+     * @return A List containing all the modified biomes and their value they are modified by
+     */
+    public static List<String> createSpawnPageBiomeTooltip(Function<Biome, Integer> biomeFunction, int originalWeight) {
```

Comment:
This is JEI specific and definitely won't be used anywhere else so maybe create own `JEIUtility` file under `itegration/jei`.